app/src/main/java/com/safetyheads/akademiaandroida/data/FirebaseConfigRepository.kt

```diff
@@ -0,0 +1,33 @@
+package com.safetyheads.akademiaandroida.data
+
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings
+import com.safetyheads.domain.entities.Config
+import com.safetyheads.domain.repositories.ConfigRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.tasks.await
+
+class FirebaseConfigRepository: ConfigRepository {
+
+    private val remoteConfig = FirebaseRemoteConfig.getInstance()
+    private val configSettings = FirebaseRemoteConfigSettings.Builder()
+        .setFetchTimeoutInSeconds(60)
+        .build()
+
+    override suspend fun getConfig(): Flow<Config> = flow {
+        remoteConfig.setConfigSettingsAsync(configSettings)
```

Comment:
Here you should use `.await()` (and you can omit result) to wait for async execution